_includes/code/components/search-result-code.html

```diff
@@ -1,11 +1,12 @@
 {% capture code_preview %}
-<h6>Without metadata</h6>
+<h6>Without Metadata</h6>
 <div class="usa-grid usa-search-result">
   <div class="usa-width-one-whole">
     <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam ut ligula pulvinar elit.</p>
   </div>
 </div>
-<h6>With metadata</h6>
+
+<h6>With Metadata</h6>
```

Comment:
@anuragmundada-reisys - Go ahead and delete this one as well. Sorry, should have been more explicit in my first comment. :)